man/man5/zfs-module-parameters.5

```diff
@@ -2623,6 +2623,22 @@ to limit potential SLOG device abuse by single active ZIL writer.
 Default value: \fB786,432\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBzio_deadman_log_all\fR (int)
+.ad
+.RS 12n
+If non-zero, the zio deadman will produce debugging messages (see
+\fBzfs_dbgmsg_enable\fR) for all zios, rather than only for leaf
+zios possessing a vdev. This is meant to be used by developers to gain
+diagnostic information for hang conditions which don't involve a mutex
+or other locking primitive; typically conditions in which a thread in
+the zio pipeline as looping indefinitely.
```

Comment:
nit: s/as/is/ looping indefinitely?

Reply:
@dweeezil can you fix this small typo, then this is ready to go.

Reply:
This has been fixed in eb3740c6496b6c88d591d010ae493c3d6af77e5c.